File: Cpp/beecrowd_1943.cpp
#include <stdio.h>

int main() {

  int x;
  scanf("%d", &x);
  int rank[] = {0, 1, 3, 5, 10, 25, 50, 100};
  for (int i = 0; i < (sizeof(rank) / sizeof(int) - 1); ++i) {
    if (x > rank[i] && x <= rank[i + 1])
      x = rank[i + 1];
  }
  printf("Top %d\n", x);
  return 0;
}

File: Python/beecrowd_1943.py
x = int(input())
rank = (0,1,3,5,10,25,50,100)
for i in range(len(rank)-1):
    if (x > rank[i] and x <= rank[i+1]):
        x = rank[i+1]    
print("Top %d" %x)
